### PR TITLE
Feat: Add the countdown account in the Hero component

### PR DESCRIPTION
**Pull request by LUISEDOCCOR**
### Before: 🚀
image
------
image

#### When Afor is live 👑:
image
#### When afor is not live 😥:
image
------
image

### CODE REVIEW

**Comment by Zyruks** [comment]
Bastante cool queda, agregaría un text-shadow para que no se pierda tanto el texto con el fondo.

**Comment by LUISEDOCCOR** [comment]
Sorry, I had not considered the component when you are live. I'll review the conflicts and check that. Thanks! 💫

**Comment by afordigital** [comment]
Thank u a lot! :3

File: src/components/Countdown/Countdown.tsx [hunk 1]
```diff
@@ -30,22 +49,12 @@ export const Countdown = () => {
...
-      <section className="text-center">
-        <p className="mb-4 text-2xl">Minutos</p>
-        <span className="font-bold text-6xl">{formatNumber(timeLeft.minutes)}</span>
-      </section>
-      <section className="text-center">
-        <p className="mb-4 text-2xl">Segundos</p>
-        <span className="font-bold text-6xl">{formatNumber(timeLeft.seconds)}</span>
-      </section>
+      {time.map(({ name, label }, index) => (
+        <section key={index} className="text-center text-shadow-sm">
```
Comment (MarcosNASA) [inline src/components/Countdown/Countdown.tsx:53]: 🟢 The `name` would likely be a better key.

**Review by MarcosNASA — COMMENTED** [review]
Looks mergeable to me! 🌮 🌯

**Pull request merged**